ci/scripts/flink-bench.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+echo "==========================================="
+echo "start the flink bench instance to run the benchmark"
+echo "==========================================="
+aws ec2 start-instances --instance-ids i-029fdf626052dcdaf
```

Comment:
I think we should make a docker image to run flink+nexmark so we don't have to start an aws instance in every build.

Reply:
Also, if we use a fixed instance, we cannot run in parallel.

Reply:
We will not run the flink benchmark so often so running it in parallel not be the issue. Also, we want to keep the fixed instance for benchmark so that we can come back and login to the instance for debugging incase of any failure.